app/controllers/concerns/two_factor_authenticatable.rb

```diff
@@ -45,10 +45,8 @@ def authenticate_user
 
   def handle_second_factor_locked_user(type)
     analytics.track_event(Analytics::MULTI_FACTOR_AUTH_MAX_ATTEMPTS)
-
-    sign_out
-
     render 'two_factor_authentication/shared/max_login_attempts_reached', locals: { type: type }
+    sign_out
```

Comment:
This order cannot be changed. The `sign_out` was specifically moved before the render to allow the user to remain on the account locked page for the duration of the lockout period. See: https://github.com/18F/identity-idp/pull/804

I verified that this change breaks that functionality, which implies that our JS test is not robust.

To test:
1. Sign in
2. Enter the wrong OTP 3 times in a row
Expected Result: you stay on the account locked page and can see the entire countdown
Actual Result: you are redirected to the home page before 1 minute has elapsed.